_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,230 @@
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" onclick="hide()" value="Hide updating content" />
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+#### Passed Example 4
+
+The text content of the `span` element automatically updates multiple times without user intervention. An [instrument][] is available to change the update frequency. The rule is not applicable to the `p` element because it has a child (the `span` element) whose content updates.
+
+```html
+<body onload="startUpdates()">
+	<p>Random number: <span id="target">1</span></p>
+	<label for="interval">Content update frequency (seconds):</label>
+	<input type="text" id="interval" />
+	<input type="button" onclick="changeFrequency(document.getElementById('interval').value)" value="Change frequency" />
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+### Failed
+
+#### Failed Example 1
+
+The text content of the `span` element automatically updates multiple times without user intervention. There is no [instrument][] to stop, pause, hide or alter the frequency of the automatic updates.
+
+```html
+<body onload="startUpdates()">
+	<p>Random number: <span id="target">1</span></p>
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+#### Failed Example 2
+
+The text content of the `span` element automatically updates multiple times without user intervention. A button is available to stop the automatic updates, but the button only stops the updates while it is focused.
+
+```html
+<body onload="startUpdates()">
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" onfocus="stopUpdates()" onblur="startUpdates()" value="Stop updates" />
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+The text content automatically updates but only as a result of the user activating a button on the page.
+
+```html
+<body>
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" id="control" onclick="startUpdates()" value="Start updates" />
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+#### Inapplicable Example 2
+
+The automatically updating text content is the only content in the document.
+
+```html
+<body onload="startUpdates()">
+	<span id="target">1</span>
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+#### Inapplicable Example 3
+
+The document does not have text content that updates automatically.
+
+```html
+<body>
+	<p>
+		The W3C Web Accessibility Initiative (WAI) develops standards and support materials to help you understand and
+		implement accessibility.
+	</p>
+</body>
+```
+
+#### Inapplicable Example 4
+
+The color of the element is updated, but not its `innerText` property.
```

Comment:
Interesting example... arguably this is blinking content, maybe? I think this should have a stop button on it. I don't think I want to get into the conversation of whether or not this satisfies the SC.